esrally/track/params.py

```diff
@@ -580,6 +580,21 @@ def percent_completed(self):
         return self.current_bulk / self.total_bulks
 
 
+class ForceMergeParamSouce:
```

Comment:
Can you please document the new parameter in https://github.com/elastic/rally/blob/master/docs/track.rst#force-merge?

IMHO it should be the first parameter in the list and the docs should say something like:

```
* ``index`` (optional, defaults to the indices defined in the ``indices`` section or ``_all`` if no indices are defined there): The name of the index that should be force-merged.
```

Reply:
Updated the docs.